### ISSUE TITLE
v4 Typo in dashboard_new.py:  pubKey = _generatePrivateKey(private_key) => pubKey = _generatePublicKey(private_key)

### ISSUE THREAD

**Issue by shuricksumy** [donaldzou/WGDashboard#268]
**Describe The Problem**
There is an typo in dashboard_new.py:  
pubKey = _generatePrivateKey(private_key) => pubKey = _generatePublicKey(private_key)

**Expected Error / Traceback**
```
500 error
wgdashboard-1  |   File "/opt/wgdashboard/dashboard_new.py", line 508, in updatePeer
wgdashboard-1  |     pubKey = _generatePrivateKey(private_key)
wgdashboard-1  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
wgdashboard-1  | TypeError: _generatePrivateKey() takes 0 positional arguments but 1 was given
wgdashboard-1  |  * Detected change in '/opt/wgdashboard/dashboard_new.py', reloading
```

```
       if len(private_key) > 0:
            pubKey = _generatePrivateKey(private_key)
            if not pubKey[0] or pubKey[1] != self.id:
                return ResponseObject(False, "Private key does not match with the public key.")
```

**Comment by donaldzou** [comment]
Ah yes, thank you for letting me know :)

**Comment by donaldzou** [comment]
Fixed in 5a34f16dcf626f84dd57394f7d33623b1898ea97 :)
